### PR TITLE
Enables monitoring dashboard to show check specific icons for a more …

### PR DESCRIPTION
**Pull request by wThomas84**
…lucid overview 
Checks can be disabled by application property 'dash.iconsEnabled'
Icons have been added so far for: Datadog, Fabric, Jenkins, Jira, ExampleCheck

### CODE REVIEW

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/check/CheckService.java [hunk 1]
```diff
@@ -42,7 +46,8 @@
     private List<CheckResult> executeCheck(Check check) {
         CheckExecutor checkExecutor = executor(check);
         try {
-            return checkExecutor.executeCheck(check);
+            List<CheckResult> checkResults = checkExecutor.executeCheck(check);
+            return decorateCheckResults(check, checkResults);
```
Comment (CanardSauvage) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/check/CheckService.java]: Yo did not want to change every Executor, correct?

File: dash-example/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,2 +1,4 @@
 de.axelspringer.ideas.tools.dash.pagetitle=testapp
+dash.iconsEnabled=true
```
Comment (CanardSauvage) [inline dash-example/src/main/resources/application.properties]: What happens if the key is not present in den properties?

**Comment by CanardSauvage** [comment]
If Henning or Timo are ok with this I'm fine. Looks good!

File: dash-example/src/main/resources/application.properties (hunk 2 above)
Comment (gaffa) [inline dash-example/src/main/resources/application.properties]: It defaults to true. So setting this property is not really necessary.

**Pull request merged**